ipu/examples/jax_ipu_quickstart.ipynb

```diff
@@ -173,6 +136,34 @@
     "print(f\"Device = {r.device()}\")"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### JAX Pseudo Random Numbers generation\n",
+    "\n",
+    "Reproducible random numbers across platforms using JAX ThreeFry PRNG."
```

Comment:
Is this already in?

Reply:
The default XLA implementation of the ThreeFry algorithm is already working well on the IPU (following the fix to bitwise shift operators). Having a custom ThreeFry vertex will give us ~10x performance improvement (and faster compilation).